client/dataEntry.js

```diff
@@ -139,6 +139,24 @@ function getSolveWarnings(resultId, solveIndex) {
   return warnings;
 }
 
+function updateWarningContents () {
+  let data = Template.currentData();
+  let selectedResultId = selectedResultIdReact.get();
+  let warnings = getSolveWarnings(selectedResultId, data.index);
+  let $warningIcon = template.$('.solve-time-warning-icon');
```

Comment:
I think you'd need to pass it in from wherever you call `updateWarningContents`, although I'm not sure the place where we're currently calling `updateWarningContents` makes sense.